tekton/ci/bindings.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: tekton-ci-webhook-pull-request
+  namespace: tektonci
+spec:
+  params:
+  - name: buildUUID
+    value: $(header.X-GitHub-Delivery)
```

Comment:
The problem with this is that all CI Jobs triggered by this event get the same buildUUID, so the logs get mixed up, like in http://logs.dogfooding.tekton.dev/?buildid=f46f3900-5edb-11ea-9990-2f5693adf309&namespace=tektonci
We could append something job-specific to the UUID, but that would break the format expected by the log app. Perhaps we could extend the log app to support that,

Reply:
cc @sbwsg

Reply:
Yeah, totally agree.  I'd be happy to update the log viewer app to accept a `taskName` or something to disambiguate the log output.